data.sql

```diff
@@ -40,3 +40,27 @@ UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
 SELECT weight_kg FROM animals;
 COMMIT;
 SELECT weight_kg FROM animals;
+
+ALTER TABLE animals ADD CONSTRAINT speicies_id_fk FOREIGN KEY (species_id) REFERENCES species(id);
+
+ALTER TABLE animals ADD owner_id INT;
+ALTER TABLE animals ADD CONSTRAINT owner_fk FOREIGN KEY (owner_id) REFERENCES owners;
```

Comment:
- [ ] Everything that creates or change the structure of a table should be saved in the schema.sql file. Let's try to work on this.